packages/api-plugin-accounts/src/resolvers/Query/filterSearchAccounts.js

```diff
@@ -0,0 +1,31 @@
+import getPaginatedResponse from "@reactioncommerce/api-utils/graphql/getPaginatedResponse.js";
+import wasFieldRequested from "@reactioncommerce/api-utils/graphql/wasFieldRequested.js";
+
+/**
+ * @name Query/accounts
```

Comment:
Is there a reason this has to be a separate endpoint rather than functionality added on to the existing queries?

Reply:
The existing query endpoint is using a fixed input structure like so
` const { groupIds, notInAnyGroups } = input;`

The new filter has an entirely different input format to handle multiple conditions. So I assume it will break any existing implementation of the query if we change it.

Reply:
Can we not extend it while not breaking the existing?

Reply:
I can see the logic of having consistent naming of the endpoint across collections but I also think it's confusing to have two endpoints that do basically the same thing but differently. What do you suggest to resolve this?

Reply:
Would you be okay to use 'filterXYZ' as the go-forward endpoint across all collections and mark the old ones as 'deprecated'? If so, we can add the new filter to all collections where there is something similar currently in place.

Else, as you mentioned initially, we have to update the existing query end-points.  We can start & test with a simple one like accounts (to confirm graphql does not create issues) and then implement that for the remaining 3 (customers/orders/products). In this case also, we may have to implement this update to other collections where there is something similar currently in place.

Reply:
I think I'm fine with the "create new and deprecate" old approach

Reply:
Ok, I shall find other collections that are using 'filter' like query and add new filter along with the promotions-filter ticket

Reply:
No, let's just stick with the three for now